config/routes.rb

```diff
@@ -53,6 +53,16 @@
       post '/' => 'users/sessions#create', as: :user_session
       get '/active' => 'users/sessions#active'
 
+      get '/account_reset/request' => 'account_reset/request#show'
+      post '/account_reset/request' => 'account_reset/request#create'
+      get '/account_reset/cancel' => 'account_reset/cancel#cancel'
+      get '/account_reset/report_fraud' => 'account_reset/report_fraud#update'
+      get '/account_reset/confirm_request' => 'account_reset/confirm_request#show'
+      get '/account_reset/delete_account' => 'account_reset/delete_account#show'
+      delete '/account_reset/delete_account' => 'account_reset/delete_account#delete'
+      get '/account_reset/confirm_delete_account' => 'account_reset/confirm_delete_account#show'
+      post '/account_reset/send_notifications' => 'account_reset/send_notifications#update'
```

Comment:
If this isn't a user-facing endpoint, can we put it under the `/api/` namespace?